modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/dto/v1_0/converter/ObjectFieldDTOConverter.java

```diff
@@ -77,8 +80,25 @@ public ObjectField toDTO(
 				state = objectField.isState();
 				system = objectField.getSystem();
 				type = ObjectField.Type.create(objectField.getDBType());
+
+				setListTypeDefinitionExternalReferenceCode(
+					() -> {
+						if (objectField.getListTypeDefinitionId() == 0) {
+							return StringPool.BLANK;
+						}
+
+						ListTypeDefinition listTypeDefinition =
+							_listTypeDefinitionLocalService.
+								fetchListTypeDefinition(
+									objectField.getListTypeDefinitionId());
+
+						return listTypeDefinition.getExternalReferenceCode();
```

Comment:
@murilostodolni should we check if `listTypeDefinition` is null?

Reply:
Hi @carolmariaabb in this case i think that not necessary. Because is toDTO, is mandatory has a listTypeDefinitionId in database. What do you think?